templates/parachain/README.md

```diff
@@ -144,8 +145,16 @@ Download the `polkadot` (and the accompanying `polkadot-prepare-worker` and `pol
 export PATH="$PATH:<path/to/binaries>"
 ```
 
+
 #### Update `zombienet-omni-node.toml` with a valid chain spec path
 
+To simplify the process of using the parachain-template with zombienet and Omni Node, we've added a pre-configured
+development chain spec (dev_chain_spec.json) to the parachain template. The zombienet-omni-node.toml file of this
+template points to it, but you can update it to an updated chain spec generated on your machine. To generate a
+chain spec refer to [staging-chain-spec-builder](https://crates.io/crates/staging-chain-spec-builder)
+
+Then make the changes in the network specification like so:
+
```

Comment:
I recalled we added this file for the `parachain-template`: https://github.com/paritytech/polkadot-sdk/blob/master/templates/parachain/README.docify.md, which should help with keeping in sync the README.md with code sections from parachain template components.

To make README.md changes for parachain template now we'll need to update the `README.docify.md` exactly as you updated the README.md, and then run (which will regenerate the README.md):

```bash
cd templates/parachain && cargo check --features generate-readme
```

LMK if you encounter issues with this.

Reply:
Hey @iulianbarbu I am done with the feedback